i18n/management/commands/publish_i18n.py

```diff
@@ -51,10 +51,13 @@ def publish_models(self):
                         list(obj.publish(silent=True))
                     # Temporary hack to turn off PDF generation while still directing users to the translated pdfs if they exist
                     if language_code in settings.LANGUAGE_GENERATE_PDF and hasattr(obj, 'publish_pdfs'):
-                        pdf_generation_start_time = time.time()
-                        list(obj.publish_pdfs(silent=True))
-                        pdf_generation_end_time = time.time()
-                        total_pdf_generation_time += (pdf_generation_end_time - pdf_generation_start_time)
+                        try:
+                          pdf_generation_start_time = time.time()
+                          list(obj.publish_pdfs(silent=True))
```

Comment:
any reason not to put this catch in the publish_pdfs method itself?

Reply:
It doesn't seem unreasonable for the page to throw an error if the pdf is failing to generate